packages/context/src/interceptor.ts

```diff
@@ -411,66 +392,10 @@ export function invokeMethodWithInterceptors(
   return tryWithFinally(
     () => {
       const interceptors = invocationCtx.loadInterceptors();
+      const targetMethodInvoker = () => invocationCtx.invokeTargetMethod();
+      interceptors.push(targetMethodInvoker);
```

Comment:
Have you considered overwriting GenericInterceptor's `invokeTarget` method instead of pushing a new interceptor to the list?

If we are not overwriting `invokeTarget`, then what is the point of having such extension point in the `GenericInterceptor` class?

Reply:
I removed invokeTarget.